### DISCUSSION TITLE
Cross-Reference to API Documentation when using MyST Parser

### DISCUSSION THREAD

**Discussion by michaelweinold** [executablebooks/meta#1169] [Q&A] [answered]
I am using the MyST parser (v3) and the [sphinx-autoapi](https://github.com/readthedocs/sphinx-autoapi) package in [our documentation](https://github.com/brightway-lca/brightway-documentation).

For instance, I would like to link to [this function](https://docs.brightway.dev/en/latest/api/bw2data/project/index.html#bw2data.project.ProjectManager.copy_project) in our documentation from a markdown page.

Unfortunately, I did not find anything in [the MyST documentation](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html) on how to cross-refence to specific parts of the auto-generated API documentation.

The [Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects) mentions the `:py:` and `:obj:` roles, but neither is documented for MyST.

Help would be greatly appreciated!

**Comment by chrisjsewell** [comment, marked as answer]
Heya, maybe check out https://myst-parser.readthedocs.io/en/latest/syntax/code_and_apis.html#documenting-whole-apis

But yes any role can be used similar to rst, just swap the colons `:` for braces `{`

**Comment by michaelweinold** [comment]
Hey Chris! Many thanks - a simple 

```
{py:obj}`bw2data.project.ProjectManager.copy_project`
```

did the trick for us.

Native MyST markdown support in your autodoc2 also sounds pretty neat - I'll give it a try soon.
